weatherman.py

```diff
@@ -0,0 +1,109 @@
+"""
+Weatherman
+This module generates yearly reports on weather data collected from  Dec 1996 - May 2011.
```

Comment:
Module does not depend on what year data is from. Avoid misleading docs.

Reply:
What I meant was the reports are with respect to year, as in the reports show us temperature details of each year.
But if you think it is misleading, I'll take it out.
Done, please review.

Reply:
I got what you meant. Changed it now.